includes/pages/admin_active.php

```diff
@@ -229,14 +229,30 @@ function admin_active()
             }
         }
 
+        $user_worklogs = $usr->worklogs()
+            ->with(['user', 'creator'])
+            ->get();
+        $shifts = Shifts_by_user($usr->id, auth()->can('user_shifts_admin'));
```

Comment:
```suggestion
        $shifts = Shifts_by_user($usr->id);
```
the permission part can be ignored here as the data is not used.

In general it would be nice to have an SQL statement here that just fetches the users shifts as the `Shifts_by_user` function adds additional informations via joins which makes the response slower (esp. as its run for every user).